### PR TITLE
Fix 400 on ninja badges index

### PR DESCRIPTION
**Pull request by ruioliveira02**

### CODE REVIEW

File: lib/bokken/uploaders/emblem.ex [hunk 1]
```diff
@@ -31,10 +31,6 @@ defmodule Bokken.Uploaders.Emblem do
 
   # Provide a default URL if there hasn't been a file uploaded
   def default_url(_version, _scope) do
-    base_url() <> "/images/default_badge.png"
-  end
-
-  defp base_url do
-    Application.fetch_env!(:waffle, :asset_host)
+    "/images/default_badge.png"
```
Comment (ruilopesm) [inline lib/bokken/uploaders/emblem.ex:34]: Why this change?

**Review by danielsp45 — APPROVED** [review]

File: lib/bokken/uploaders/emblem.ex (hunk 1 above)
Comment (ruioliveira02) [reply lib/bokken/uploaders/emblem.ex:34]: Bacause the default badge will not be stored remotely

Comment (ruilopesm) [reply lib/bokken/uploaders/emblem.ex:34]: That way, all badges will have the default badge image 🤔

**Review by ruilopesm — APPROVED** [review]
